packages/desktop/views/dashboard/vesting/Vesting.svelte

```diff
@@ -56,11 +66,24 @@
                     : undefined,
         },
     ]
-
-    let modal: Modal
-    let timeUntilNextPayout = DEFAULT_EMPTY_VALUE_STRING
     $: $selectedAccountVestingPayouts, (timeUntilNextPayout = getTimeUntilNextPayout())
 
+    onMount(() => {
+        getMinRequiredStorageDeposit()
+        areOutputsReadyForConsolidation()
+    })
+
+    function areOutputsReadyForConsolidation(): void {
+        $selectedAccount
+            .prepareConsolidateOutputs({ force: false, outputThreshold: 2 })
+            .then(() => (hasOutputsToConsolidate = true))
+            .catch(() => (hasOutputsToConsolidate = false))
+    }
+
+    async function getMinRequiredStorageDeposit() {
```

Comment:
same as `getMinRequiredStorageDeposit`, I think it would be better to modify the variable inside a `then` instead of making the function asynchronous